packages/liferay-npm-scripts/src/jest/mocks/liferay.js

```diff
@@ -0,0 +1,113 @@
+/**
+ * © 2019 Liferay, Inc. <https://liferay.com>
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+const authToken = 'default-mocked-auth-token';
+
+/**
+ * Event support APIs on the `Liferay` object inherited from `A.Attributes`
+ *
+ * Defines:
+ * 	- Liferay.detach
+ * 	- Liferay.fire
+ * 	- Liferay.on
+ *
+ * https://github.com/liferay/liferay-portal/blob/master/modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/events.js#L66
+ * https://yuilibrary.com/yui/docs/api/classes/Attribute.html
+ */
+const events = {
+	/**
+	 * https://yuilibrary.com/yui/docs/api/files/event-custom_js_event-target.js.html#l372
+	 */
+	detach: jest.fn(),
+
+	/**
+	 * https://yuilibrary.com/yui/docs/api/files/event-custom_js_event-target.js.html#l700
+	 */
+	fire: jest.fn(),
+
+	/**
+	 * https://yuilibrary.com/yui/docs/api/files/event-custom_js_event-target.js.html#l227
+	 */
+	on: jest.fn(),
+
+	/**
+	 * https://yuilibrary.com/yui/docs/api/files/event-custom_js_event-target.js.html#l136
+	 */
+	once: jest.fn()
+};
+
+/**
+ * Contains a fallback/dummy implementation of `Liferay.Language.get`. In practice, this call is rewritten in a ServerFilter, so runtime calls to
+ * `Liferay.Language.get` should not be found in production code. A better match for the real behaviour would be a babel plugin to rewrite calls
+ * to the API with their "translated" value.
```

Comment:
If by this you mean replacing them with a string (not the actual string, as that seems like too much work), then I agree with you.

Reply:
yup!